diff.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"bytes"
+	"log"
+
+	"github.com/pmezard/go-difflib/difflib"
+)
+
+var cmdDiff = &Command{
+	Usage: "diff",
+	Short: "shows the diff between current and previously saved set of godeps",
+	Long: `
+Shows the difference, in a unified diff format, between the 
+current set of godeps and those generated on a 
```

Comment:
I suggest s/godeps/dependencies/ because the current set isn't actually saved anywhere yet. As a user I don't conceptualize it as "godeps" until it's written to the file.